lib/backend/dynamo/dynamodbbk.go

```diff
@@ -475,6 +535,26 @@ func (b *DynamoDBBackend) createKey(fullPath string, val []byte, ttl time.Durati
 	return nil
 }
 
+// encrypt helper encrypts the record Value
+func (r *record) encrypt(encryptedKey []byte, dataKey []byte, keyID string) error {
+	r.EncryptedKey = encryptedKey
+	r.KeyID = keyID
+	block, err := aes.NewCipher(dataKey)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	ciphertext := make([]byte, aes.BlockSize+len(r.Value))
+	iv := ciphertext[:aes.BlockSize]
+	if _, err := io.ReadFull(rand.Reader, iv); err != nil {
+		return trace.Wrap(err)
+	}
+
+	stream := cipher.NewCFBEncrypter(block, iv)
```

Comment:
Using CFB here means that a block of the plaintext can be corrupted / partially controlled and no checksum means this can't be proven to be a hostile change... but given that the stored values are other keys, does it even matter? (they'd just stop working)

I'm not necessarily saying that AEAD/GCM or similar should be used instead (although, why not?), but a note about why this implementation doesn't care about authentication could be useful.